Client/Frontend/Browser/Onboarding/Welcome/ReleaseNotesView.swift

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveShared
+
+struct ReleaseNotesView: View {
+    @Environment(\.colorScheme) var colorScheme: ColorScheme
+
+    var dismiss: (() -> Void)?
+
+    var body: some View {
+        VStack {
+            Image(uiImage: #imageLiteral(resourceName: "brave-vpn-ad-top-shimmer"))
+            .resizable()
+            .aspectRatio(contentMode: .fit)
+            HStack {
+                Spacer()
+                Button {
+                    dismiss?()
+                } label: {
+                    Image(uiImage: #imageLiteral(resourceName: "privacy-everywhere-exit-icon"))
+                }
+                .padding(.trailing, 10)
+                .padding(.top, 10)
+            }
+            Text("Brave new releases")
+            Image(uiImage: #imageLiteral(resourceName: "brave-vpn-ad-background"))
+            .resizable()
+            .aspectRatio(contentMode: .fit)
+            .overlay(imageOverlay, alignment: .center)
+            .padding()
+            VStack(alignment: .leading, spacing: 8.0) {
+                Text("• One line of release notes")
+                Text("• Another line of release notes")
+            }
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .padding()
+        }
+        .frame(maxWidth: 450)
+        .background(Color(.braveBackground))
+        .cornerRadius(5.0)
+    }
+    
+    private var imageOverlay: some View {
+        VStack {
+            HStack {
+                Text("Firewall")
+                    .font(Font(UIFont.systemFont(ofSize: 27.0, weight: .bold)))
+                    .rotationEffect(Angle(degrees: -9.57), anchor: .bottomLeading)
+                    .foregroundColor(Color.white)
+                
+                Text("+")
```

Comment:
make it `verbatim` temporarily for both so it does not get pulled for translation

Reply:
Removed at https://github.com/brave/brave-ios/pull/4508/commits/79cef7b33427ec27bc534f34e6076574bd198213